pom.xml

```diff
@@ -41,18 +41,21 @@
     <properties>
         <version.checkstyle.plugin>2.9.1</version.checkstyle.plugin>
         <version.org.jboss.byteman>3.0.1</version.org.jboss.byteman>
-        <version.org.jboss.logging.jboss-logging>3.1.4.GA</version.org.jboss.logging.jboss-logging>
+        <version.org.jboss.logging.jboss-logging>3.2.1.Final</version.org.jboss.logging.jboss-logging>
         <version.org.jboss.logging.jboss-logging-processor>1.1.0.Final</version.org.jboss.logging.jboss-logging-processor>
         <version.org.jboss.logmanager>1.5.2.Final</version.org.jboss.logmanager>
+        <version.org.jboss.modules>1.4.3.Final</version.org.jboss.modules>
         <version.org.jboss.marshalling>1.4.3.Final</version.org.jboss.marshalling>
-        <version.org.jboss.remoting>4.0.5.Final</version.org.jboss.remoting>
+        <version.org.jboss.remoting>5.0.0.Beta4</version.org.jboss.remoting>
```

Comment:
It looks like Remoting is pulling in a snapshot of Elytron (that's probably my mistake).  We need to ensure that we aren't pulling in any transitives; could you please add an explicit dependency on a stable Elytron?

Snapshot dependencies break CI, so that's the primary reason, but it's probably good to keep things as stable as possible as a general principle.